.env.production

```diff
@@ -1 +1,3 @@
 RAILS_MAX_THREADS=2
+# https://island94.org/2025/01/ruby-thread-contention-simply-gvl-queuing
+RUBY_THREAD_TIMESLICE=10
```

Comment:
(Mostly unrelated, but I noticed that this was in `.env.production.local` with my secret env vars, but there's no reason for it to be secret, so I'm moving it here.)